modules/router-store/src/router_store_module.ts

```diff
@@ -35,6 +42,7 @@ import {
   SerializedRouterStateSnapshot,
 } from './serializers/default_serializer';
 import { MinimalRouterStateSerializer } from './serializers/minimal_serializer';
+import { _ACTIVE_RUNTIME_CHECKS } from '../../store/src/tokens';
```

Comment:
`_ACTIVE_RUNTIME_CHECKS` was meant to be private to `store`, so this is actually illegal.

looking for your thoughts @timdeschryver if the right approach to having this warning includes exposing the injection token for the active runtime checks

Reply:
I went ahead and exposed it, YOLO

Reply:
Somehow I missed this.
I think this is fine, at least I don't see how we can expose it otherwise.